console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/assets/DeviceAssetsPanel.java

```diff
@@ -369,6 +369,12 @@ public void handleEvent(BaseEvent be) {
             radioFalse.setValue(true);
             radioGroup.setOriginalValue(radioFalse);
         }
+        if(channel.getModeEnum().equals(GwtDeviceAssetChannelMode.WRITE)) {
+            radioTrue.setValue(null);
```

Comment:
What means setting value to null?

Reply:
Hi @muros-ct.

It means if channel is only with write permission, than value in radio button is null, and user can not see value, but he can chose false or true option.

Reply:
Ok, I understand now. It is disabling radio button functionality. None of buttons is selected, neither true or false. Hack.